src/patternfly/components/Menu/menu-check-input.hbs

```diff
@@ -0,0 +1,5 @@
+<input class="pf-c-menu__check-input{{#if menu-check-input--modifier}} {{menu-check-input--modifier}}{{/if}}"
```

Comment:
I wonder if instead of this, we could have a `__check` element that holds a standalone check component. IMO it would be great if we could use the check component everywhere so all checkbox enhancements, themes, etc are available in components. Seems like updating all of our use of single checkboxes in components would be a good breaking change update.